### PR TITLE
Add 2 new fonts (Jost and Tilt Warp)

### LINKED ISSUE #12: Add fonts to project
By default, tailwind has only 3 classes for applying font-family.

We should figure out what fonts we will use for project and more custom classes for fonts.

Two possible options:

- use default fonts like `Helvetica` or `Tahoma` or `Verdana` etc. This will require just some changes in tailwind config file.

- use downloaded fonts, like Google fonts. This will require additional step for importing fonts.

I will leave the above choice and choice of particular fonts to the assignee.

Good starting point for research:

https://tailwindcss.com/docs/font-family

For completing this issue, add couple of custom classes for font-family and apply it to 2 or 3 html elements on our page.
In PR original post brifly specify what fonts did you add and where on the page it was applied.

This issue is not about choosing the whole font scheme for app, we just need something to start with, and all the fonts will be easy to fix/edit/add in the future.

### PR DESCRIPTION
**Pull request by Sakoutecher**
# Description

I had 2 new google fonts, first i add google fonts links into the head of the index.html.
After this i configure the fonts in the config file of tailwind, now to use those 2 fonts, type `font-title` for the Tilt Warp font, and type `font-paragraph` for the Jost font.
And to finish i apply those new fonts on all the text in the index.html to see.

Fixes #12 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Chore items (this includes basic clean up of files, or updates to packages)
- [ ] Updates to documentation

## Checklist

- [X] I have a descriptive title for my PR
- [X] I have linked the issue to this PR. [Instruction how to link issues to pull requests here](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [X] I have run the project locally and reviewed the code changes
- [X] My changes generate no new warnings

### CODE REVIEW

**Review by Dorfieeee — CHANGES_REQUESTED** [review]
Hey!

I like how you try to empathize the more important information using `Tilt Warp` and other text using `Jost`. But I don't like that there is another font for the other numbers and text. 

Only this small section has 3 different fonts and I think it is a bit messy.
image

Moreover, when we add another HTML, we would need to explicitly set the `font-paragraph` class.

Could you change the config so the `Jost` is default font for the whole document and therefore you can remove all the `font-paragraph` classes inside HTML?

Or do you think it's better the way it's done now?

File: index.html [hunk 1]
```diff
@@ -65,26 +78,28 @@
                 <div id="icon" class="md:w-3/4 max-w-[12rem]"></div>
               </div>
               <div class="flex items-center gap-2">
-                <span id="temp-min"
-                  >min.<span class="font-medium text-lg ml-1" data-value>0</span
+                <span id="temp-min font-paragraph"
```
Comment (Dorfieeee) [inline index.html:81]: When I run the code, I get an error in the console and some details don't show up and not even the weather image.

image

File: index.html [hunk 2]
```diff
@@ -65,26 +78,28 @@
...
+                  >min.<span class="font-paragraph text-lg ml-1" data-value
+                    >0</span
                   ><span data-unit>&#176;C</span></span
                 >
                 <span class="mx-1 border-r-2 h-3/4"></span>
-                <span id="temp-max"
-                  >max.<span class="font-medium text-lg ml-1" data-value>0</span
+                <span id="temp-max font-paragraph"
```
Comment (Dorfieeee) [inline index.html:87]: Same mistake here. I did not check for others...

File: index.html (hunk 1 above)
Comment (Sakoutecher) [reply index.html:81]: My bad i didnt see it

**Comment by Sakoutecher** [comment]
My bad for those mistakes, i didnt see it, i will correct that tomorrow ! I will also adjust the section with 3 fonts.
I think it's better to keep the font-paragraph, it's more maintanable.
Let me know what you think about.

**Comment by Dorfieeee** [comment]
@Sakoutecher so you want to add `font-paragraph` to all other elements that are not `font-title`?
In that case, it makes more sense to set `Jost` globally and then select only elements you want with `font-title`.

**Comment by Sakoutecher** [comment]
As you want ! I will do thaï.

**Comment by Dorfieeee** [comment]
It's not about what I want but what makes more sense. You decided to use those fonts and how to use them in the code. I only question your actions.

**Comment by Sakoutecher** [comment]
Hello, how are you ? 
So i remove all the font-paragraph on all the text.
I had font-global to the tailwind config file and i put it on the body to had it on the page.
In the console i have an error from updateWeather.js, i dont think its from me.
Let me know if it's ok for you !

**Review by Dorfieeee — APPROVED** [review]
Hey,

I don't see any warnings any longer because you fixed them by removing all the `font-paragraph` from the HTML.

You made a mistake where you typed the class name inside id attribute:
```
<span id="temp-max font-paragraph"`
```

I believe there is a different way of implementing global fonts in tailwind but this is working and I don't see any reason why changing it now for an app of this size.

LGTM 👍🏻

**Comment by Dorfieeee** [comment]
@Sakoutecher make sure that your changes don't cause any errors next time.

This is not just a checkbox exercise!
image

Imagine you contribute to a large open source project or pushing code to your company repo.. what would the people think when you pushing code with errors and claiming that it was not your fault.. 

If you like to be part of this project beyond this issue, you can join our Discord server, you can find invite link in the README file.

**Pull request merged**